tests/e2e/suites/basic/basic_suite_test.go

```diff
@@ -0,0 +1,34 @@
+package basic
+
+import (
+	"testing"
+
+	// . "github.com/onsi/ginkgo/v2"
+	// . "github.com/onsi/gomega"
+
```

Comment:
```suggestion
```

Reply:
I left this in as an example of what to add when you start adding tests. Happy to have it removed if you like  😄

Reply:
Na, that's a pretty good reason. Keep it! 😄